susemanager-utils/susemanager-sls/salt/certs/Rocky8.sls

```diff
@@ -0,0 +1 @@
+RedHat8.sls
```

Comment:
This does not work, it has to have the whole content of the RedHat8.sls file in it, which means:
```
/etc/pki/ca-trust/source/anchors/RHN-ORG-TRUSTED-SSL-CERT:
  file.managed:
    - source:
      - salt://certs/RHN-ORG-TRUSTED-SSL-CERT

update-ca-certificates:
  cmd.run:
    - name: /usr/bin/update-ca-trust extract
    - runas: root
    - onchanges:
      - file: /etc/pki/ca-trust/source/anchors/RHN-ORG-TRUSTED-SSL-CERT
```

Reply:
Apologies - how do I generate this in github?

Reply:
Hmm... good question, I would have done it via a bash with `ln -s` but I either can't find a function directly in Github for that

Reply:
Fixed.